src/backend/gl/src/lib.rs

```diff
@@ -489,6 +473,9 @@ impl CommandQueue {
             self.state.vao = true
         }
 
+        // Reset indirect draw buffer
+        unsafe { gl.BindBuffer(gl::DRAW_INDIRECT_BUFFER, 0) };
```

Comment:
that would surely confuse some older driver, we need to gate this by a device feature